tensorflow/compiler/xla/service/gpu/horizontal_loop_fusion_test.cc

```diff
@@ -505,6 +505,73 @@ TEST_F(HorizontalLoopFusionTest, IterativeHorizontalFusion) {
   EXPECT_EQ(total_fusion_instrs, 2);
 }
 
+TEST_F(HorizontalLoopFusionTest, TraversalOrder) {
+  auto module = ParseAndReturnVerifiedModule(R"(
```

Comment:
Also would prefer the actual type.

Reply:
Expanding this one is a bit lengthy. (i..e, `StatusOr<std::unique_ptr<VerifiedHloModule>>`)

Skip it but I expand all other reasonable candidates in the file though.